third_party/brotli/c/enc/hash_longest_match_quickly_inc.h

```diff
@@ -248,6 +248,7 @@ static BROTLI_INLINE void FN(FindLongestMatch)(
       }
     }
   }
+  BROTLI_UNUSED(min_score);  /* Calm down MSVC. */
```

Comment:
Shouldn't be needed anymore ?

Reply:
This one is added here on purpose: https://lbshell-internal-review.git.corp.google.com/c/third_party/brotli/+/99350. I guess it's still needed. cc @yuying-y to confirm.

Reply:
If you remove it and builds don't issue a warning, it's okay to take it out. It may have been done against an older MSVC compiler. It's fairly minor though and can be left in to get this landed.

Reply:
Already cherry-pick the submodule commit and see this. Maybe get this PR land first.